tensorflow/python/keras/backend.py

```diff
@@ -149,7 +149,7 @@ def cast_to_floatx(x):
 
   Example:
   ```python
-      >>> from keras import backend as K
+      >>> from tensorflow.keras import backend as K
       >>> K.floatx()
```

Comment:
This is the docstring for `cast_to_floatx`, so I guess this should be `K.cast_to_floatx`

Reply:
Hi, I think the K.floatx makes sense in context with the rest of the example.

```python
      >>> from tensorflow.keras import backend as K
      >>> K.floatx()
      'float32'
      >>> arr = numpy.array([1.0, 2.0], dtype='float64')
      >>> arr.dtype
      dtype('float64')
      >>> new_arr = K.cast_to_floatx(arr)
      >>> new_arr
      array([ 1.,  2.], dtype=float32)
      >>> new_arr.dtype
      dtype('float32')
  ```

The K.floatx is to show what the default float type is for the Keras backend, so that when we later call K.cast_to_floatx, we can show it was casted to the default float type.

Reply:
You are right, of course. Sorry!